conans/client/cmd/uploader.py

```diff
@@ -360,15 +360,16 @@ def _package_files_to_upload(self, pref, policy, the_files, remote):
         remote_snapshot = self._remote_manager.get_package_snapshot(pref, remote)
 
         if remote_snapshot:
-            remote_manifest, _ = self._remote_manager.get_package_manifest(pref, remote)
-            local_manifest = FileTreeManifest.loads(load(the_files["conanmanifest.txt"]))
+            if CONAN_MANIFEST in remote_snapshot.keys():
```

Comment:
``if CONAN_MANIFEST in remote_snapshot:``

key contained in dict type doesn't require ``keys()``